.github/workflows/rust.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         sys:
         - os: ubuntu-latest-16-cores
           target: x86_64-unknown-linux-gnu
-        - os: ubuntu-latest-16-cores
+        - os: ubuntu-jammy-16-cores-arm64
           target: aarch64-unknown-linux-gnu
         - os: macos-latest
           target: x86_64-apple-darwin
```

Comment:
On GitHub macos-latest is arm64 (equiv to macos-14), whilst macos-12 was the last stable x86 harware. Technically there is also macos-13, but on GitHub macos-13 was never in the "latest" chain.

We could run the x86 build on macos-12, we do that for the `binaries.yml` workflow already. Then the build willrun on native hardware. Functionally it shouldn't matter which we use because macos-14 can run x86 binaries. The only reason to do it would be if we saw better performance by not emulating.

Taking a look at the builds, it doesn't look like we need it. At least the builds both take the same amount of time.

<img width="542" alt="Screenshot 2024-11-06 at 10 59 27 PM" src="https://github.com/user-attachments/assets/34a52a1b-7fe9-4769-984c-8088a3e582db">